priv/repo/migrations/00000000000001_create_user.exs

```diff
@@ -9,7 +9,6 @@ defmodule Org.Repo.Migrations.CreateUser do
       add :login, :string
       add :company, :string
       add :location, :string
-      add :created_at, :string
```

Comment:
Oh, guess I should've asked first. How about `github_created_at` for a little more clarity?

It doesn't need a too in-depth review, it's a WIP. Forgot to put that in the PR title.